nerdtree_plugin/fs_menu.vim

```diff
@@ -225,10 +225,12 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if !empty(treenode)
-        if has("osx")
-            let stat_cmd = 'stat -f "%z" '
-        else
-            let stat_cmd = 'stat -c "%s" '
+        if has("unix")
+            let s:uname = system("uname")
+            let stat_cmd = 'stat -c "%s" ' 
+            if s:uname == "Darwin\n"
```

Comment:
Change this operator to `=~?` and remove the `\n`. The case-insensitive regex will help future-proof the if statement. See `:h expr-=~?`.